test.js

```diff
@@ -34,7 +34,7 @@ test('extend', t => {
     [tagKey]    : randomTagName,
     [randomAttr]: randomValue
   }
-  const parse = parseTags.extend(randomTagName, {tagKey})
+  const parse = extend(randomTagName, {tagKey})
```

Comment:
This is technically an API change, but IMO an improvement. Now `parseTags.extend()` is no longer a function; it must be imported as its own module.

We could add `.extend()` to `parse()`’s prototype, but that’d have to happen in JS and is no longer supported by the module system.